tools/serverpod_cli/lib/src/generator/code_generation_collector.dart

```diff
@@ -27,7 +27,7 @@ class CodeGenerationCollector extends CodeAnalysisCollector {
   String toString() {
```

Comment:
Question: When do we use this toString method? Could we remove it?

Reply:
Not sure actually